pom.xml

```diff
@@ -272,37 +273,7 @@
       </plugin>
       <plugin>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.6.1</version>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
-          <fork>true</fork>
-          <compilerArguments>
-            <Xmaxerrs>10000</Xmaxerrs>
-            <Xmaxwarns>10000</Xmaxwarns>
-          </compilerArguments>
-          <annotationProcessorPaths>
-            <path>
-              <groupId>org.checkerframework</groupId>
-              <artifactId>checker</artifactId>
-              <version>2.8.0</version>
-            </path>
-          </annotationProcessorPaths>
-          <annotationProcessors>

```

Comment:
Please make sure that the buildfile runs the checkers.

Reply:
wasn't running checkers because i didn't do a mvn clean.  doh!

merge error is fixed